traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -296,11 +294,5 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	api.CreateChangeLogRawTx(api.ApiChange, "Snapshot of CRConfig performed for "+cdn, inf.User, inf.Tx.Tx)
-	http.Redirect(w, r, "/tools/flash_and_close/"+url.PathEscape("Successfully wrote the CRConfig.json!"), http.StatusFound)
-}
-
-func writePerlHTMLErr(w http.ResponseWriter, r *http.Request, tx *sql.Tx, logErr error, err error) {
-	log.Errorln(logErr.Error())
-	tx.Rollback()
-	http.Redirect(w, r, "/tools/flash_and_close/"+url.PathEscape("Error: "+err.Error()), http.StatusFound)
+	http.Redirect(w, r, client.API_v13_CDNs+"/"+cdn+"/snapshot", http.StatusFound)
```

Comment:
So this just feels weird to me. Including the _client_ which is based on the TO api seems almost cyclical? Don't we want to simply redirect akin to:
```
http.Redirect(w, r, "/api/1.1/cdns/"+cdn+"/snapshot", http.StatusSeeOther)
```
I know we're just trying to use a const version here, but I dunno. Should we make it a const in this package so we aren't relying on a v1 client? I do like the idea, once removing v1 api and client that this will automatically be picked up, but something just itches the wrong way for me here.

Reply:
> Should we make it a const in this package so we aren't relying on a v1 client?

If we weren't getting rid of API v1, we might put it in [/lib/go-tc](https://github.com/apache/trafficcontrol/tree/master/lib/go-tc), since that's the canonical place to put shared constants.

Another place to put this kind of thing is [/traffic_ops/toclientlib](https://github.com/apache/trafficcontrol/tree/master/traffic_ops/toclientlib) (added in #5552), though IMO that should be somewhere in /lib, too.

Reply:
Yeah referring the client seemed a bit weird to me as well, but as Zach mentioned, we are getting rid of API v1.